pkg/utils/time/time.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2024 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package time contains time specific functions and variable to make it to overwrite for unit tests
+package time
+
+import "time"
+
+var Now = time.Now
```

Comment:
had to do something similar in this [PR](https://github.com/vmware-tanzu/tanzu-cli/pull/819/files#diff-e7351021bcc81adaa7ed3b1f735852841a7bad195f45b957c71aca058f04d9a4R23), but putting this in its own package is cleaner.